packages/documentation/.storybook/preview.js

```diff
@@ -49,4 +68,9 @@ export const parameters = {
       date: /Date$/,
     },
   },
+  darkMode: {
```

Comment:
```suggestion
  darkMode: {
    dark: { ...theming.dark, ...postTheme.dark },
    light: { ...theming.light, ...postTheme.light },
```
We need to provide a dark and a light postTheme and override the default themes with it.
Please see: https://storybook.js.org/addons/storybook-dark-mode -> Section "Configuration"